File: app/src/main/java/com/ndipatri/solarmonitor/providers/solarUpdate/dto/solaredge/GetOverviewResponse.java

package com.ndipatri.solarmonitor.providers.solarUpdate.dto.solaredge;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class GetOverviewResponse {

    @SerializedName("overview")
    @Expose
    private Overview overview;

    public Overview getOverview() {
        return overview;
    }

    public void setOverview(Overview overview) {
        this.overview = overview;
    }

}

File: app/src/main/java/com/ndipatri/solarmonitor/providers/solarUpdate/dto/solaredge/GetOverviewResponse.kt
package com.ndipatri.solarmonitor.providers.solarUpdate.dto.solaredge

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class GetOverviewResponse {

    @SerializedName("overview")
    @Expose
    var overview: Overview? = null

}
